UKHO.PeriodicOutputService/UKHO.AdmiraltyInformationOverlay.Fulfilment/Services/FulfilmentDataService.cs

```diff
@@ -435,6 +442,44 @@ private async Task DownloadAioAncillaryFilesAsync(string batchId, FormattedWeekN
             _logger.LogInformation(EventIds.AioAncillaryFilesDownloadCompleted.ToEventId(), "Downloading of AIO base exchange set ancillary files completed | {DateTime} | _X-Correlation-ID : {CorrelationId}", DateTime.UtcNow, CommonHelper.CorrelationID);
         }
 
+        /// <summary>
+        /// Update the content of serial.aio file to fix
+        /// https://dev.azure.com/ukhydro/Abzu/_workitems/edit/220298
+        /// </summary>
+        /// <param name="fileDetails"></param>
+        /// <param name="downloadPath"></param>
+        /// <param name="cdType"></param>
+        /// <param name="weekNumber"></param>
+        /// <returns></returns>
+        /// <exception cref="FulfilmentException"></exception>
+        private async Task UpdateSerialAioFile(List<FssBatchFile> fileDetails, string downloadPath, string cdType, FormattedWeekNumber weekNumber)
```

Comment:
This is producing a warning during build because it's an async method but doesn't await anything in the method, although I don't think it matters.
```suggestion
        private void UpdateSerialAioFile(List<FssBatchFile> fileDetails, string downloadPath, string cdType, FormattedWeekNumber weekNumber)
```

Reply:
Done